tests/test_accountRequests.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+import sys, os
+path = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0,path+'/../')
+
+from nessie.accountRequests import AccountRequests
+from nessie.models.account import Account
+
+class TestAccountRequests(unittest.TestCase):
+    # copied pasted accounts main into test case
+    def test_accounts(self):
+        accReq = AccountRequests("e7131ca32d226c905701fcca6ab9ad13")
```

Comment:
I have reservations about exposing your personal API key to public github, but we can discuss